### DISCUSSION TITLE
You can change routes/click outside the beacon

### DISCUSSION THREAD

**Discussion by iamalextodoran** [gilbarbara/react-joyride#638] [Q&A] [answered]
Start the tour -> the beacon appears -> click anywhere outside the beacon (you can change routes/open menus/etc.)

Is there any way to disable clicking outside of it? Or a way to cancel the tour if the user clicks outside the beacon?

**Comment by gilbarbara** [comment, marked as answer]
hey @iamalextodoran

This library doesn't know what's happening in the rest of your app, so it can't control its own state in these conditions.

You can lift the tour state up in the component tree and change the `run` prop when some of these changes happen.
